<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportTabCapitalGain</class>
 <widget class="QWidget" name="ReportTabCapitalGain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Report Tab</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string>&lt;p&gt;On this tab, you set the basic properties of this report.&lt;/p&gt;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="lblSettlementPeriod">
       <property name="text">
        <string>Settlement period</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblTermsSeparator">
       <property name="text">
        <string>Terms separator</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" alignment="Qt::AlignRight">
      <widget class="QCheckBox" name="m_showSTLTCapitalGains"/>
     </item>
     <item row="3" column="1">
      <widget class="KMyMoneyDateInput" name="m_termSeparator" native="true">
       <property name="toolTip">
        <string>Before this date investments are counted as long-term investments.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Show short-term and long-term capital gains</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblinvestmentSum">
       <property name="text">
        <string>Sum investments</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="m_settlementPeriod">
       <property name="toolTip">
        <string>Time in days between the settlement date and the transaction date.</string>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="m_investmentSum"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Hide Totals</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="5">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="1" alignment="Qt::AlignRight">
      <widget class="QCheckBox" name="m_checkHideTotals">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer15">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KMyMoneyDateInput</class>
   <extends>QWidget</extends>
   <header>kmymoneydateinput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
